app/components/Login/login.css

```diff
@@ -17,5 +17,5 @@
 }
 
 .error {
-    color:  rgb(244, 67, 54);
+    color:  red;
```

Comment:
changing my red eh - that specific color matches the materialUI red color that comes with the framework (red500).  I think this is a different brand of red.